### DISCUSSION TITLE
请求分秒帧个人云盘支持

### DISCUSSION THREAD

**Discussion by fengguowudi** [AlistGo/alist#278] [Ideas]
https://www.mediatrack.cn/
里面带一个无限容量无限大小的个人云盘及素材库，使用的是腾讯云的oss。速度非常快。

**Comment by xhofe** [comment]
登录有数美滑块验证，token有效期只有七天，且无法刷新。

**Comment by xhofe** [comment]
Available in v2.0.5.

**Comment by wuhenge** [comment]
账户会被封禁的，我用了几天账户就被封禁了
